### ISSUE TITLE
Build an OrbitDB-based app

### ISSUE THREAD

**Issue by ArcticSpaceFox** [BuildingXwithJS/proposals#39]
Build an app that uses react as frontend framework, Bulma as CSS framework and that uses OrbitDB as datastore. everything is in Javascript. If you think a shop is too time-consuming maybe try a Blog or a file exchanger... idk. I hope you do this because I am really interested in seeing how this would be done.

**Comment by ArcticSpaceFox** [comment]
I still hope this gets chosen too! 😄

**Comment by yamalight** [comment]
I've already done more than one video on react + bulma, but OrbidDB looks pretty interesting.
So, if we turn this into purely OrbidDB thing - I think I'd be in for trying it out 🤔

**Comment by ArcticSpaceFox** [comment]
Yeah, it was just to simplify the looking and building process, also I love bulma 😂😅. But yes it is super cool, becuase making distributed realtime data that scales is not hard at all. Also the community is happy to help you out on any questions.

**Comment by ArcticSpaceFox** [comment]
Hey,
not to bother you, but maybe when you build this thing, deploy it using [Terminal](https://terminal.co).
It is free so no worrys, if not its ok too. Very cool thing to use though.

**Comment by mandaputtra** [comment]
Interesting Idea ~

**Comment by ArcticSpaceFox** [comment]
Thanks, I just wanted to fit the theme a little more. It shouldn't take too much time and the community support was amazing!!!
